mycroft/skills/intent/__init__.py

```diff
@@ -57,6 +104,9 @@ def handle_utterance(self, message):
             reply = message.reply(
                 best_intent.get('intent_type'), best_intent)
             self.emitter.emit(reply)
+            # best intent detected -> update called skills dict
+            name = self.intent_to_skill[best_intent['intent_type']]
```

Comment:
name is an artifact from before when i used the skill_name instead of skill_id, changing now : P

totally missed that list thing, will also look into it